roles/kubernetes-apps/cluster_roles/tasks/main.yml

```diff
@@ -75,6 +75,33 @@
     - node_webhook_crb_manifest.changed
   tags: node-webhook
 
+- name: Write vsphere-cloud-provider ClusterRole manifest
+  template:
+    src: "vsphere-rbac.yml.j2"
+    dest: "{{ kube_config_dir }}/vsphere-rbac.yml"
+  register: vsphere_rbac_manifest
+  when:
+    - rbac_enabled
+    - cloud_provider is defined
+    - cloud_provider == 'vsphere'
+    - kube_version | version_compare('v1.9.0', '>=')
```

Comment:
Let's be very specific with targeting here. I think `x >= v1.9.0` and `x <= 1.9.3` if I'm reading the linked kubernetes issue correctly.

Reply:
Correct. Only needed for `x >= v1.9.0` and `x <= 1.9.3`